src/async.c

```diff
@@ -276,39 +289,25 @@ mpd_async_io(struct mpd_async *async, enum mpd_async_event events)
 	return true;
 }
 
-bool
-mpd_async_send_command_v(struct mpd_async *async, const char *command,
-			 va_list args)
+static bool
+quote_vargs(struct mpd_buffer *buffer, char *start, char **end_pos,
+	    va_list args)
 {
-	size_t room, length;
-	char *dest, *end, *p;
+	char *end, *p;
 	const char *arg;
 
-	assert(async != NULL);
-	assert(command != NULL);
-
-	if (mpd_error_is_defined(&async->error))
-		return false;
-
-	room = mpd_buffer_room(&async->output);
-	length = strlen(command);
-	if (room <= length)
-		return false;
-
-	dest = mpd_buffer_write(&async->output);
 	/* -1 because we reserve space for the \n character */
-	end = dest + room - 1;
-
-	/* copy the command (no quoting, we asumme it is "clean") */
-
-	memcpy(dest, command, length);
-	p = dest + length;
+	/**
```

Comment:
Don't use Doxygen-style comments unless you're documenting an API.

Reply:
fixed in https://github.com/MusicPlayerDaemon/libmpdclient/pull/53/commits/14a9563740b652aaeae007ccc93cadda44211e9c